aziotctl/src/init.rs

```diff
@@ -334,6 +334,7 @@ fn run_inner(stdin: &mut impl Reader) -> Result<RunOutput> {
             aziot_keys: Default::default(),
             preloaded_keys: Default::default(),
             endpoints: Default::default(),
+            principal: Default::default(),
```

Comment:
CS and IS will need access to a bunch of keys.

1. CS will need access to EST cert's key and EST bootstrap's cert key in case EST is configured.

1. CS will need access to local CA's key in case local CA is configured.

1. IS will need access to device ID key in case of symmkey and X.509 provisioning.

1. IS will need access to `aziot_identityd_master_id` key (always).

Reply:
Added